src/vm/node_modules/VM.js

```diff
@@ -12064,8 +12067,6 @@ function deleteZone(uuid, log, callback)
                 'VM has local data volumes, checking other containers for '
                 + '--volumes-from');
 
-            // TODO can we safely limit to the same owner_uuid?
```

Comment:
I removed this line because I don't think we can ever reasonably limit this to owner_uuid. Standalone SmartOS often will not have an owner_uuid and we still need to check for it there.